plugins/clipboard/lang/sk.js

```diff
@@ -11,5 +11,6 @@ CKEDITOR.plugins.setLang( 'clipboard', 'sk', {
 	pasteNotification: 'Stlačte %1 na vloženie. Váš prehliadač nepodporuje vloženie prostredníctvom tlačidla v nástrojovej lište alebo voľby v kontextovom menu.',
 	pasteArea: 'Miesto pre vloženie',
 	pasteMsg: 'Vložte svoj obsah do nasledujúcej oblasti a stlačte OK.',
-	fileFormatNotSupportedNotification: 'Tento formát súboru nie je podporovaný. Môžete to skúsiť s jedným z podporovaných formátov: ${formats}.'
+	fileFormatNotSupportedNotification: 'Tento formát súboru nie je podporovaný. Môžete to skúsiť s jedným z podporovaných formátov: ${formats}.',
```

Comment:
The same here.

It would not be a problem if the new notification wasn't reversed (it no longer lists the valid formats, it now lists the invalid ones).